lib/internal/Magento/Framework/App/Test/Unit/MaintenanceModeTest.php

```diff
@@ -186,11 +192,146 @@ public function testSetAddresses()
     }
 
     /**
-     * Set single address test
+     * Set single IPv4 address test
+     *
+     * @return void
+     */
+    public function testSetSingleAddressV4Legacy()
```

Comment:
Please add the return type for the newly added and modified methods

```suggestion
    public function testSetSingleAddressV4Legacy(): void
```